js/src/components/google-ads-account-card/connect-ads/index.js

```diff
@@ -131,17 +129,10 @@ const ConnectAds = ( props ) => {
 							) }
 						/>
 					) : (
-						<AppButton
-							isSecondary
-							disabled={ ! value }
-							eventName="gla_ads_account_connect_button_click"
-							eventProps={ getEventProps( {
-								id: Number( value ),
-							} ) }
-							onClick={ handleConnectClick }
-						>
-							{ __( 'Connect', 'google-listings-and-ads' ) }
-						</AppButton>
+						<ConnectCTA
+							handleConnectClick={ handleConnectClick }
+							value={ Number( value ) }
```

Comment:
@ankitrox I don't think we need `Number` here.
```suggestion
							value={ value }
```

Reply:
Updated here : https://github.com/woocommerce/google-listings-and-ads/pull/2640/commits/8a562d15c1b5ef969841d5f18d38bb92df40ed54#diff-6048a51e5a593289afa602e53ec8f9cbd2bf257f6724a874ddc26fe35a937e7dR134